examples/compute_instance/disk_snapshot/variables.tf

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+
+variable "project_id" {
+  description = "The GCP project to use for integration tests"
+  type        = string
+}
+
+variable "region" {
+  description = "The GCP region to create and test resources in"
+  type        = string
+  default     = "us-central1"
+}
+
+variable "subnetwork" {
+  description = "The subnetwork selflink to host the compute instances in"
+}
+
+variable "nat_ip" {
+  description = "Public ip address"
+  default     = null
+}
+
+variable "network_tier" {
+  description = "Network network_tier"
+  default     = "PREMIUM"
+}
+
+variable "num_instances" {
+  description = "Number of instances to create"
+}
+
+variable "additional_disks" {
```

Comment:
This doesn't need to be a variable, just have the config inside the example. Examples don't need to be highly customizable, as they are meant to be a starting point for copy & paste.

Reply:
In the example it is needed because I'm building a local variable `instance_disks` which dynamically creates a list of disk names

Reply:
That doesn't mean it needs to be a variable though, couldn't you hardcode the local variable value?

Reply:
True. Removed the variable and also added some comments when setting up the local var